src/main/java/org/springframework/data/redis/connection/lettuce/LettuceConnectionFactory.java

```diff
@@ -1163,6 +1187,7 @@ void resetConnection() {
 		private @Nullable String clientName;
 		private Duration timeout = Duration.ofSeconds(RedisURI.DEFAULT_TIMEOUT);
 		private Duration shutdownTimeout = Duration.ofMillis(100);
+		private Duration shutdownQuietPeriod = Duration.ofMillis(100);
```

Comment:
`MutableLettuceClientConfiguration` is a legacy bridge that provides `LettuceClientConfiguration` for older clients that do not want to upgrade to provide a dedicated `LettuceClientConfiguration`. Let's keep the behavior as-is since we don't want to add features here (i.e. `getShutdownQuietPeriod()` would return `shutdownTimeout `).